modules/quickstart-public-cloud/pages/clients.adoc

```diff
@@ -17,10 +17,27 @@ You might need to add the FQDN of the server to [path]``/etc/hosts`` local resol
 Check using the [command]``hostname -f`` command with the local IP address of the server.
 ====
 
+
 Public cloud images usually do not allow SSH login with username and password, but only SSH with a certificate.
-If you want to use bootstrap from the {webui}, you will need to enable SSH login with username and password.
+If you want to use bootstrap from the {webui}, you will need to enable SSH login with username and SSH key.
 You can do this by navigating to menu:Systems[Bootstrapping] and changing the authentication method.
 
+If your cloud provider is Microsoft Azure, you can login with username and password.
+To do this, you need to allow the AzureUser to run commands as root without a password.
+To do this, open the [path]``/etc/sudoers.d/waagent`` file, and add or edit this line:
++
+----
+AzureUser ALL=(ALL) NOPASSWD: ALL
+----
+
+[WARNING]
+====
+Allowing the AzureUser to run commands as root without a password carries a security risk.
+Use this method for testing only.
```

Comment:
In this case, we should define what testing means. I'd say accessible by the test user only.

Reply:
I think "test user" is actually a little specific. What I'm trying to say is "use this to try it out, not in prod". I don't want people to think it's safer if they just create a user called "test".